openstack/ae-environment.yml

```diff
@@ -0,0 +1,89 @@
+heat_template_version: 2013-05-23
+description: Aeternity environment
+
+parameters:
+    environment:
+        type: string
+        label: Nodes Environment
+    node_count:
+        type: number
+        label: Nodes Count
+        description: Number of epoch nodes
+        default: 2
+        constraints:
+          - range: { min: 2 }
+            description: Nodes Count must be at least 2
+    node_flavor:
+        type: string
+        label: Node Instance Type
+        description: Type of instance (flavor) to deploy for the epoch node
+        default: c1.small
+        constraints:
+          - custom_constraint: nova.flavor
+    key_name:
+        type: string
+        label: Key Name
+        description: SSH key to be used for all instances
+        default: "Travis Master"
+        constraints:
+          - custom_constraint: nova.keypair
+    image_name:
+        type: string
+        label: Image Name
+        description: Image name to create node servers
+        default: "Ubuntu 16.04"
+
+resources:
+  management_security_group:
+    type: OS::Neutron::SecurityGroup
+    properties:
+      description: Allow node management from internet (ICMP, SSH, API)
+      name:
+        str_replace:
+          template: ae-%env%-management
+          params:
+            "%env%": { get_param: environment }
+      rules:
+        - { direction: ingress, remote_ip_prefix: 0.0.0.0/0, protocol: tcp, port_range_min: 3013, port_range_max: 3013}
```

Comment:
This port would be better be parametrized. No need to change in this PR though.

Reply:
I added parameter in 4dc701d